api/spec/v1.yaml

```diff
@@ -2089,6 +2089,10 @@ components:
             applicable. The meaning varies based on the transaction method.
             Usually in native denomination, ParaTime units. As a string.
           example: "100000001666393459"
+        encryption_envelope:
```

Comment:
I remember now that I saw you were asking on Slack whether people think of "the encryption envelope" as just the encryption metadata (format, key, nonce) or also the encrypted data itself. Sorry I never answered. Unlike this PR, I myself thought of it as metadata only. But that doesn't matter much, what matters is what the internet thinks, and that's ... nothing. It thinks it's a misspelling of "envelope encryption". After more sleuthing, "digital envelope" seems to be well established. In both cases, the expected contents of the envelope (which does seem to include the data, btw :)) are pretty universally understood to be the asymmetrically encrypted DEK (data encryption key; itself symmetric), and the data itself (encrypted with DEK).

Since our envelope is a little different, maybe it's best to avoid a term with such a strongly defined narrow meaning. #407 uses `EVMEncryptedData`. We could go with that ... or `evm_encrypted_contents` (to avoid having `data` inside `[...]_data`?

Reply:
discussed offline. We will use `encryption_envelope` but update the description to note the difference between our encryption envelopes and the traditional [digital envelope](https://nordvpn.com/cybersecurity/glossary/digital-envelope/)

Reply:
feels like it serves the same purpose as a digital envelope, it uses an asymmetric scheme to set up a symmetric key and it has the data encrypted with that symmetric key